src/foam/nanos/crunch/test/UCJEditBehaviourTest.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch.test',
```

Comment:
I gotta ask, if the capability list only exists in NP does this just fail when it is run in foam?

Reply:
@AlexeyGreer as @sarthak-marwaha mentioned, if the capabilities selected to test: firstname, .... are only onboarding capabilities on nanopay side, then move the test case to nanopay.  net/nanopay/crunch/test/